lib/gunit/SConscript

```diff
@@ -26,44 +26,51 @@ env.VariantDir(Dir('.').abspath + '/gtest', gtest_path + '/src')
 env.VariantDir(Dir('.').abspath + '/gmock', gmock_path + '/src')
 
 gmocklibname = 'gmock.lib'
-gtestlibname = 'gtest{d}.lib'.format(d='d' if env['TARGET_CONFIG'] =='debug' else  '')
+gtestlibname = 'gtest.lib'
```

Comment:
both release and debug libraries will not have the d suffix as per the project files.